docs/shared/getting-started/start-with-existing-project.md

```diff
@@ -12,12 +12,67 @@ Thanks to [Nx's modular architecture](/getting-started/intro), you can start wit
 Getting started is remarkably simple. You can add Nx to any existing project with a single command:
 
 ```shell
-nx init
+nx@latest init
```

Comment:
been bitten by the npm cache too many times so figured I'd add it just to be safe. but can remove if we feel strongly